k6/vu.go

```diff
@@ -14,16 +14,19 @@ import (
 	k6lib "go.k6.io/k6/lib"
 	k6metrics "go.k6.io/k6/metrics"
 	"gopkg.in/guregu/null.v3"
-
-	"github.com/grafana/xk6-browser/common"
 )
 
-type mockVU struct {
+// VU is a k6 VU instance.
+type VU struct {
```

Comment:
I think this should still be called `MockVU` as that is what is being created in the functions below

Reply:
For the record, explained [here](https://github.com/grafana/xk6-browser/pull/315#discussion_r876230016).